driver-moab/main/seq_map_mod.F90

```diff
@@ -112,7 +112,7 @@ subroutine seq_map_init_rcfile( mapper, comp_s, comp_d, &
     if (mct_gsmap_Identical(gsmap_s,gsmap_d)) then
        if(.not.skip_match) call seq_map_mapmatch(mapid,gsmap_s=gsmap_s,gsmap_d=gsmap_d,strategy="copy")
 
-       if (mapid > 0 .and. .not. skip_match) then
+       if (.not. skip_match .and. mapid > 0 ) then
```

Comment:
What's the difference here?

Reply:
Just switched it to match other "if" clauses where skip_match is checkd first.